cmake/FindQtRuntime.cmake

```diff
@@ -0,0 +1,53 @@
+# Find a compatible Qt version
+# Sets COCKATRICE_QT_VERSION_NAME -- Example values: Qt5, Qt6
+
+FIND_PACKAGE(Qt6 6.2.3 COMPONENTS
+        Concurrent
```

Comment:
I don't like how we require all components here, we should just require the minimum for each compile target so you can avoid unnecessary things for your server etc